src/platforms/hubspot/schemas.ts

```diff
@@ -627,3 +627,19 @@ export const hubspotAssociationLabelOutputSchema = z.object({
 export type HubspotAssociationLabelOutput = z.infer<
   typeof hubspotAssociationLabelOutputSchema
 >;
+
+export const hubspotAccessTokenOutputSchema = z.object({
+  token: z.string(),
```

Comment:
should any of these be optional/nullish?

Reply:
nope, the api docs mention that it's always returned!